CONTRIBUTING.md

```diff
@@ -153,6 +153,18 @@ get it merged. You can also post in the `#docs` [Slack
 channel](https://knative.slack.com/) to get input on your ideas or find areas to
 contribute before creating a PR.
 
+### Common Troubleshooting issues for PRs
```

Comment:
I'd think `DEVELOPMENT.md` is probably a good place for this.  I don't feel super strongly, but my spidey sense seems to think this file is largely boilerplate across the repos (I think some may even redirect folks to a common one in community?)...

Reply:
Sounds good. I'll remove this section.